mlir/include/mlir/Dialect/LLVMIR/Transforms/Passes.td

```diff
@@ -11,6 +11,11 @@
 
 include "mlir/Pass/PassBase.td"
 
+def LLVMAddComdats : Pass<"llvm-add-comdats", "::mlir::ModuleOp"> {
+  let summary = "Add comdats to linkonce and linkonce_odr functions";
```

Comment:
Can you add a description about the context for this pass, more info about the invariant, why it's used, etc.
(this will be generated in the doc on the website!)